src/RMGGeneratorDecay0.cc

```diff
@@ -43,4 +50,53 @@ void RMGGeneratorDecay0::GeneratePrimaries(G4Event* event) {
   fDecay0G4Generator->GeneratePrimaries(event);
 }
 
+void RMGGeneratorDecay0::SetMode(std::string mode) {
+  try {
+    fDecayMode = RMGTools::ToEnum<RMGGeneratorDecay0::DecayMode>(mode, "decay mode");
+  } catch (const std::bad_cast&) { return; }
+  if (fDecayMode == DecayMode::k2vbb) {
+    nuclide = "Ge76";
+    dbd_mode = 4;
+    dbd_level = 0;
+  } else if (fDecayMode == DecayMode::k0vbb) {
+    nuclide = "Ge76";
+    dbd_mode = 1;
+    dbd_level = 0;
+  } else {
+    RMGLog::Out(RMGLog::error, "Unknown decay mode");
+  }
+  seed = static_cast<G4int>(G4UniformRand() * std::numeric_limits<G4int>::max());
+  debug = false;
+
+  bxdecay0_g4::PrimaryGeneratorAction::ConfigurationInterface& configInt = fDecay0G4Generator
+                                                                               ->GrabConfiguration();
+
+  configInt.reset_mdl();
+  configInt.decay_category = "dbd";
+  configInt.nuclide = nuclide;
+  configInt.seed = seed;
+  configInt.dbd_mode = dbd_mode;
+  configInt.dbd_level = dbd_level;
+  configInt.debug = debug;
+  fDecay0G4Generator->SetConfigHasChanged(true);
+}
+
+void RMGGeneratorDecay0::DefineCommands() {
+
+  // NOTE: SetUnit(Category) is not thread-safe
+
+  fMessenger = std::make_unique<G4GenericMessenger>(
+      this,
+      "/RMG/Generator/Decay0/",
+      "Commands for controlling the BxDecay0 generator"
+  );
+
+  fMessenger->DeclareMethod("Mode", &RMGGeneratorDecay0::SetMode)
```

Comment:
We could maybe call this 
```suggestion
  fMessenger->DeclareMethod("Process", &RMGGeneratorDecay0::SetMode)
```